app/src/middleware/carboneCopy.js

```diff
@@ -212,12 +212,18 @@ const initializeApiTracker = (app, basePath) => {
 };
 
 const security = (req, res, next) => {
-  const operation = req._carboneOp;
-  if (operation) {
-    if (operation.isGenerator) {
-      return keycloak.protect(`${clientId}:GENERATOR`)(req, res, next);
-    } else {
-      return keycloak.protect()(req, res, next);
+  // allow docs and the api spec to be accessible by non-authenticated users.
+  // everything else should be authenticated
+  if (!['/docs','/api-spec.yaml'].includes(req.url)) {
```

Comment:
Might be good to have as a constants list rather than hardcoded, but also not too concerned since this is going to be the only thing we don't restrict.

Reply:
yeah, have a complicated structure for all the important operations, didn't want to mess with that or come up with a separate one for these two paths... KISS